infra/Makefile.toml

```diff
@@ -0,0 +1,597 @@
+docker compose -f ./fendermint/infra/docker-compose.yml up -d
+"""
+
+[tasks.testnet-down]
+dependencies = [
+  "testnet-docker-compose-down",
+  "network-rm"
+]
+
+[tasks.testnet-docker-compose-down]
+script = """
+docker compose -f ./fendermint/infra/docker-compose.yml down
+"""
+
+[tasks.testnet-init]
+dependencies = [
+  "testnet-clear",
+  "testnet-network-create",
+  "cometbft-pull",
+  "testnet-mkdir",
+  "testnet-cometbft-init",
+  "testnet-mkdir",
+]
+
+[tasks.testnet-network-create]
+env = { "NETWORK_NAME"="${NETWORK_NAME}"}
+extend = "network-create"
+
+[tasks.testnet-clear]
+script="""
+echo clearing all IPC data
+rm -rf ${BASE_TESTNET_DIR}
+"""
+
+[tasks.testnet-mkdir]
+script="""
+echo creating directories: $BASE_DIR $FM_DIR $CMT_DIR
+mkdir -p $BASE_TESTNET_DIR
+mkdir -p $BASE_TESTNET_DIR/out
+
+mkdir -p $BASE_TESTNET_DIR/node0
+mkdir -p $BASE_TESTNET_DIR/node1
+mkdir -p $BASE_TESTNET_DIR/node2
+mkdir -p $BASE_TESTNET_DIR/node3
+
+mkdir -p $BASE_TESTNET_DIR/node0/fendermint
+mkdir -p $BASE_TESTNET_DIR/node1/fendermint
+mkdir -p $BASE_TESTNET_DIR/node2/fendermint
+mkdir -p $BASE_TESTNET_DIR/node3/fendermint
+
+mkdir -p $BASE_TESTNET_DIR/node0/cometbft
+mkdir -p $BASE_TESTNET_DIR/node1/cometbft
+mkdir -p $BASE_TESTNET_DIR/node2/cometbft
+mkdir -p $BASE_TESTNET_DIR/node3/cometbft
+"""
+
+[tasks.testnet-cometbft-init]
+dependencies = ["testnet-cometbft-init-node0", "testnet-cometbft-init-node1", "testnet-cometbft-init-node2", "testnet-cometbft-init-node3"]
+
+[tasks.testnet-cometbft-init-node0]
+extend = "cometbft-init"
+env = { "CMD" = "init", "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_TESTNET_DIR}/node0/cometbft", "CMT_CONTAINER_NAME" = "cometbft-node0", "FLAGS" = "-a STDOUT -a STDERR --rm"}
+
+[tasks.testnet-cometbft-init-node1]
+extend = "cometbft-init"
+env = { "CMD" = "init", "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_TESTNET_DIR}/node1/cometbft", "CMT_CONTAINER_NAME" = "cometbft-node1", "FLAGS" = "-a STDOUT -a STDERR --rm"  }
+
+[tasks.testnet-cometbft-init-node2]
+extend = "cometbft-init"
+env = { "CMD" = "init", "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_TESTNET_DIR}/node2/cometbft", "CMT_CONTAINER_NAME" = "cometbft-node2", "FLAGS" = "-a STDOUT -a STDERR --rm"  }
+
+[tasks.testnet-cometbft-init-node3]
+extend = "cometbft-init"
+env = { "CMD" = "init", "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_TESTNET_DIR}/node3/cometbft", "CMT_CONTAINER_NAME" = "cometbft-node3", "FLAGS" = "-a STDOUT -a STDERR --rm"  }
+
+[tasks.testnet-fendermint-config]
+dependencies = [
+  "testnet-config-newgenesis",
+  "testnet-config-peers",
+  "testnet-config-newkey",
+  "testnet-config-newaccount",
+  "testnet-config-newgateway",
+  "testnet-share-genesis"
+]
+
+[tasks.testnet-config-newgenesis]
+cwd = "./target/release"
+script = """
+./fendermint genesis --genesis-file $BASE_TESTNET_DIR/genesis.json \
+      new --chain-name testnet --base-fee ${BASE_FEE} --timestamp ${TIMESTAMP}
```

Comment:
`--chain-name` here is different than in the `docker-compose` file.

Reply:
Fixed